tools/distrib/check_copyright.py

```diff
@@ -173,7 +214,12 @@ def log(cond, why, filename):
     if m:
         pass
     elif 'DO NOT EDIT' not in text:
-        log(1, 'copyright missing', filename)
+        if args.fix:
+            text = license_text + '\n\n' + text
```

Comment:
`license_text` already has one `\n`, if we only want 1 blank one, we need just 1 additonal `\n` here.

Reply:
We're using `'\n'.join()` before so two are needed for a blank line.
Anyway, this is no longer relevant in the updated CL.